raven/processes/wps_hydrobasins_shape_selection.py

```diff
@@ -136,10 +139,10 @@ def _handler(self, request, response):
 
                 feat = json.loads(agg.to_json())['features'][0]
                 response.outputs['feature'].data = json.dumps(feat)
-                response.outputs['upstream_ids'].data = json.dumps(up['HYBAS_ID'].tolist())
+                response.outputs['upstream_ids'].data = json.dumps(up['id'].tolist())
```

Comment:
I guess these are the IDs needed to extract WFS features afterward ? Won't this break code using specifically the HYBAS_ID ?

Reply:
Maybe there is no such code, but this is the kind of things that should be clearly highlighted in the PR description.

Reply:
Right. Yes, I'll be sure to be clear about changes like this in the future.

As it stands, there is no code that makes use of the `HYBAS_ID`. Since the database rests solely accessible via WFS, the data is more easily queried using the GeoServer address and this `id` (when written to file is called the `gml_id`). I felt the change was necessary as the `upstream_ids` were not the `HYBAS_ID`'s either, instead being the WFS `id`'s. Now everything is consistent.